src/Tes/Utilities/ConnectionStringUtility.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Text;
+
+namespace Tes.Utilities
+{
+    public class ConnectionStringUtility
+    {
+        public string GetPostgresConnectionString(
+            string postgreSqlServerName, 
+            string postgreSqlTesDatabaseName, 
+            string postgreSqlTesDatabasePort, 
+            string postgreSqlTesUserLogin, 
+            string postgreSqlTesUserPassword,
+            string postgreSqlServerNameSuffix = ".postgres.database.azure.com",
+            string sslMode = "VerifyFull")
+        {
+            if (string.IsNullOrEmpty(postgreSqlServerName))
```

Comment:
nit: we should agree on a consistent approach to validate the args - I've been using the newer approach of ArgumentNullException.ThrowIfNull.  Note: I don't have a strong preference, and I'd be okay to change my approach if that will drive consistency.

Reply:
Done